lib/services/http-service.js

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016, EMC, Inc.
+// Copyright 2015, EMC, Inc.
```

Comment:
Well, this is just update to this file. It was originally created on 2015. I guess the copyright should reflect the first time the file created ?